scalafix-tests/input/src/main/scala/tests/ExplicitResultTypesRefinement.scala

```diff
@@ -27,11 +27,15 @@ object ExplicitResultTypesRefinement {
   def tparam[T <: CharSequence](e: T) = new Serializable {
     val results: List[Int] = List(e.length())
   }
-  //todo: in 2.13, the type found is AnyRef
-  //  val access = new Serializable {
-  //    private val results: List[Int] = List.empty
-  //    protected val results2: List[Int] = List.empty
-  //  }
+  val access = new Serializable {
+    private val results: List[Int] = List.empty
+    protected val results2: List[Int] = List.empty
+  }
+  val product = new Product {
+    override def productArity: Int = ???
+    override def productElement(n: Int): Any = ???
+    override def canEqual(that: Any): Boolean = ???
+  }
```

Comment:
maybe you can add a `new Serializable with Product` since the fix seems to be related to that?

Reply:
good Idea.